test/linter/test-notes.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+const chalk = require('chalk');
+const parser = require('node-html-parser');
+const { HtmlValidate } = require('html-validate');
+const { VALID_ELEMENTS } = require('../utils.js');
+
+const validator = new HtmlValidate();
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ *
+ * @typedef {'disallowed' | 'attrs' | 'attrs_a' | 'invalid' | 'doublespace'} HTMLErrorType
+ *
+ * @typedef {object} HTMLError
+ * @property {HTMLErrorType} errortype The type of error
+ * @property {string} relPath The identifier of the feature
```

Comment:
What do the `relPath` variable names mean? Relative path? If so, relative to what?